vi/src/main/resources/Messages_vi.properties

```diff
@@ -0,0 +1,91 @@
+app_name = tR Chunk Browser
+export = Xuất bản
+add = Thêm
+choose_output_folder = Chọn thư mục đầu ra
+choose_wav_file = Chọn file WAV
+drop_here = Thả thư mục tR hoặc file WAV vào đây
+merge = Gộp
+split = Tách
+remove = Xoá
+select_all = Chọn tất cả
+clear_all = Xóa tất cả
+file_already_imported = Tập tin này đã được nhập
+done_exporting = Xuất dữ liệu hoàn tất
+export_error = Không thể xuất bản. Sửa đổi metadata (dữ liệu thông tin) và thử lại
+import_error = Không thể nhập file
+not_wav_file = File không phải kiểu âm thanh WAV
+convert_folder_title = Chuyển đổi thư mục tR
+convert_folder_message = Tạo bản sao thư mục và tách ra tất cả file thu âm từng đoạn thành file thu âm từng câu?
+cancel = Huỷ
+take = Ghi âm
+ok = OK
+yes = Đồng ý
+no = Không
+export_in_progress = Đang trong quá trình xuất bản...
+localization_restart_app = Vui lòng khởi động lại ứng dụng để thay đổi có hiệu lực
+gen = Genesis
```

Comment:
Would you translate book names?

Reply:
I'm pending on that and some more parts I have not done yet